src/Input/Silk.NET.Input.Sdl/SdlKeyboard.cs

```diff
@@ -18,7 +19,7 @@ public SdlKeyboard(SdlInputContext ctx)
         public string Name { get; } = "Silk.NET Keyboard (via SDL)";
         public int Index { get; } = 0;
         public bool IsConnected { get; } = true;
-        public IReadOnlyList<Key> SupportedKeys { get; } = new ReadOnlyCollectionListAdapter<Key>(_keyMap.Values);
+        public IReadOnlyList<Key> SupportedKeys { get; } = _keyMap.Values.Where(x => x != Key.Unknown).ToArray();
```

Comment:
```suggestion
        public IReadOnlyList<Key> SupportedKeys { get; }
            = _keyMap.Values.Where(static x => x != Key.Unknown).ToArray();

```

Reply:
Thought about distinct. If you think so too, sounds good.